regoservice/regocheck.go

```diff
@@ -3,31 +3,55 @@ package regoservice
 import (
 	"context"
 	"fmt"
+	"os"
+	"strings"
 
 	"github.com/open-policy-agent/opa/rego"
 )
 
-const module = `package postee
+const (
+	module = `package postee
 
 default allow = false
 
 allow {
 %s
 }
 `
+	defaultPathToRegoFilters = "./rego-filters"
+)
 
-func DoesMatchRegoCriteria(input interface{}, rule string) (bool, error) {
+func DoesMatchRegoCriteria(input interface{}, files []string, rule string) (bool, error) {
+	ctx := context.Background()
+	r := &rego.Rego{}
+	pathToRegoFilters := defaultPathToRegoFilters
 
-	if rule == "" {
-		return true, nil //no rule defined - any input allowed
-	}
+	if len(files) != 0 {
+		if os.Getenv("REGO_FILTERS_PATH") != "" {
+			pathToRegoFilters = os.Getenv("REGO_FILTERS_PATH")
+		}
+		if !strings.HasSuffix(pathToRegoFilters, "/") {
+			pathToRegoFilters += "/"
+		}
+		for i, file := range files {
+			if !strings.HasPrefix(file, pathToRegoFilters) {
+				files[i] = pathToRegoFilters + file
```

Comment:
use `filepath.Join()` to join parts of the path

Reply:
also move all path merging logic to separate function

Reply:
done